tests/includes/juju.sh

```diff
@@ -189,3 +197,12 @@ wait_for() {
         sleep 5
     done
 }
+
+introspect_controller() {
+    local name
+
+    name=${1}
+
+    juju ssh -m controller 0 "source /etc/profile.d/juju-introspection.sh && juju_engine_report" > "${TEST_DIR}/${name}-juju_engine_report.txt"
```

Comment:
We have to force the inclusion of the script here, because it's not loaded unless you do a full login i.e. `juju ssh -m controller 0` vs `juju ssh -m controller 0 bash`. The later doesn't load anything in `profile.d`...